bedrock/firefox/templates/firefox/browsers/mobile/android.html

```diff
@@ -23,6 +23,22 @@
   {% endif %}
 {% endblock %}
 
+{% block experiments %}
+  {% if switch('firefox-android-sms-experiment', ['en-US']) and country_code == "US" %}
+    {{ js_bundle('firefox-android-sms-experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% if variation == '1' and country_code == "US" %}
+  {% set s2d_include = 'firefox/browsers/mobile/includes/s2d-android-email.html' %}
+{% elif variation == '2' and country_code == "US" %}
+  {% set s2d_include = 'firefox/browsers/mobile/includes/s2d-android-sms.html' %}
+{% elif variation == '3' and country_code == "US" %}
+  {% set s2d_include = 'firefox/browsers/mobile/includes/s2d-android-qr.html' %}
+{% else %}
+  {% set s2d_include = 'firefox/browsers/mobile/includes/s2d-android.html' %}
+{% endif %}
```

Comment:
**Suggestion:** In the same vein from my PR you could simplify this with a nested if statement with `country_code == 'US'` on the outerblock